compiler/rustc_target/src/spec/armv6k_nintendo_3ds.rs

```diff
@@ -0,0 +1,45 @@
+use crate::spec::{LinkArgs, LinkerFlavor, PanicStrategy, RelocModel, Target, TargetOptions};
+
+/// A base target for Nintendo 3DS devices using the devkitARM toolchain.
+///
+/// Requires the devkitARM toolchain for 3DS targets on the host system.
+
+pub fn target() -> Target {
+    let mut pre_link_args = LinkArgs::new();
+    pre_link_args.insert(
+        LinkerFlavor::Gcc,
+        vec![
+            "-specs=3dsx.specs".to_string(),
+            "-g".to_string(),
+            "-march=armv6k".to_string(),
+            "-mtune=mpcore".to_string(),
```

Comment:
Do the `-g`, `-march` and `-mtune` arguments do anything here? `gcc` here does not compile any code, so intuitively unless there's some link-time codegen going on, these shouldn't be doing anything.

Reply:
I knew they were needed but didn't check them one at a time. Turns out `mtune` is the only one required of the 3 during linking. Thanks for the help.